gnomad_qc/v3/annotations/generate_freq_data.py

```diff
@@ -161,9 +146,15 @@ def main(args):
                 else mt.meta.project_meta.project_subpop,
                 # NOTE: TGP and HGDP labeled populations are highly specific and are stored in the project_subpop meta field
             )
+            freq_meta=[{**x, **{"subset": subset}} for x in hl.eval(mt.freq_meta)]
```

Comment:
I missed this, but this needs to be modified because there is no `subset` variable `subsets` and is a list of subsets to include